packages/opal-common/requires.txt

```diff
@@ -1,8 +1,8 @@
 aiohttp
 click
-cryptography
+cryptography>=35.0.0
 gitpython
-loguru==0.6.0
-pyjwt[crypto]==2.1.0
+loguru
```

Comment:
maybe we should set a minimum version

Reply:
Updated; now the specifier is `loguru>=0.6.0,<1`